tflint/README.md

```diff
@@ -0,0 +1,49 @@
+# tflint 
+
+This package is a wrapper of the ![tflint](https://github.com/terraform-linters/tflint) project, enabling it to be natively executed from the before and after hooks. The `tflint/tflint.go` uses the MPL 2.0 license, due to the license of tflint.
```

Comment:
```suggestion
This package allows us to embed [tflint](https://github.com/terraform-linters/tflint) in Terragrunt, enabling it to be natively executed from the before and after hooks without having to install `tflint` separately. Since `tflint` is licensed with MPL, we are required to let you know where you can find its source code: https://github.com/terraform-linters/tflint
```

Per Q8 of MPL, as far as I can tell, using `tflint` this way (unmodified) does not change the license of Terragrunt. It just requires us to tell users where they can find the source code of the MPL project: https://www.mozilla.org/en-US/MPL/2.0/FAQ/